lib/src/page_list_viewport_gestures.dart

```diff
@@ -442,7 +636,29 @@ class DeprecatedPanAndScaleVelocityTracker {
       return;
     }
 
+    // Check that the two swipes consequtively considered for repeated
+    // swiping acceleration are collinear
+    if (_isPossibleRepeatedAcceleratedSwipe && !((_previousLaunchVelocity.dy * velocity.dy).isNegative)) {
+      // Proceed to increase the momentum simulation initial boost to
+      // scroll faster
+      _numberOfRepeatedAcceleratedSwipes++;
+      // If the user makes tiny slow scrolls, which get deccelerated by
+      // constants due to their scale, they wont't be accelerated for the first 3 of such swipes.
+      if (_numberOfRepeatedAcceleratedSwipes > 2) {
+        _momentumSimulationInitialVelocityIncreaseScalar =
+            _repeatedSwipeVelocityIncreaseScalar(_numberOfRepeatedAcceleratedSwipes);
+        _dragIncreaseScalar = _repeatedSwipeDragIncreaseScalar(_numberOfRepeatedAcceleratedSwipes);
+      }
+    } else {
+      _resetRepeatedAccelerationTracking();
+    }
+
     _launchVelocity = velocity;
+    // Updating a repeated swipe tracking parameter
```

Comment:
This comment doesn't clarify the code beneath it. Please explain the significance of `_launchVelocity.distance > 0` and then explain what it means to set that flag to `true`.

Also, isn't `_launchVelocity.distance` the same as `speed` from up above?